src/zenml/utils/mlstacks_utils.py

```diff
@@ -625,7 +625,7 @@ def deploy_mlstacks_stack(
     declare("Stack successfully deployed.")
 
     if not no_import_stack_flag:
-        declare(f"Importing stack '{stack_name}' into ZenML...")
+        declare(f"Importing stack '{stack_name}' into ZenML..")
```

Comment:
😄

Reply:
I have no idea how this ended up in there, must have been some search and replace :D